.rubocop.yml

```diff
@@ -181,3 +181,4 @@ Style/Next:
 # Keep things fitting nicely on github
 Metrics/LineLength:
   Max: 116
+  AllowURI: true
```

Comment:
@gaganawhad @joeosburn that's what this card is for :) https://trello.com/c/soninm8B/1690-enforce-line-length-limit